diff --git a/requirements.txt b/requirements.txt
index fd4934b1..1de0ed58 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,6 @@ scikit-image>=0.19
 scikit-learn
 dask
 dask-jobqueue
-opencv-python
 xarray>=2022.11.0
 statsmodels
 altair
diff --git a/setup.py b/setup.py
index 8bd1691d..ddc75b04 100644
--- a/setup.py
+++ b/setup.py
@@ -92,8 +92,6 @@ setuptools.setup(
     # extras_require={
     #     'test': ['pytest-runner', 'pytest'],
     # },
-    setup_requires=["pytest-runner"],
-    tests_require=['pytest'],
     cmdclass=versioneer.get_cmdclass(),
     # scripts=["plantcv-train.py", "plantcv-utils.py", "plantcv-workflow.py"]
 
