diff --git a/requirements.txt b/requirements.txt
index 966b9653..b0adb0f2 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -8,5 +8,4 @@ scikit-learn
 plotnine
 dask
 dask-jobqueue
-opencv-python
 statsmodels
diff --git a/setup.py b/setup.py
index 3a22ecb2..793ecdd0 100644
--- a/setup.py
+++ b/setup.py
@@ -109,8 +109,6 @@ setuptools.setup(
     # extras_require={
     #     'test': ['pytest-runner', 'pytest'],
     # },
-    setup_requires=["pytest-runner"],
-    tests_require=['pytest'],
     cmdclass=versioneer.get_cmdclass(),
     scripts=["plantcv-train.py", "plantcv-utils.py", "plantcv-workflow.py"]
 
